### DISCUSSION TITLE
Berry debug reset

### DISCUSSION THREAD

**Discussion by trlafleur** [arendst/Tasmota#14944] [Ideas]
It would be very convenient to have a Berry reset button next to the Run button in the Berry console, to allow one to reset Berry without needing to reboot ESP32/Tasmota during debugging of Berry code...

thanks

**Comment by s-hadinger** [comment]
That's a very interesting suggestion. I need to check that there are no nasty side effects. We could indeed delete the current Berry VM and start a new one. Let check if it would work.
